### DISCUSSION TITLE
Lightdm

### DISCUSSION THREAD

**Discussion by GoodBoyNeon** [catppuccin/catppuccin#1843] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Maybe

**Additional context 📷 (optional)**
Maybe something similar to the port of [sddm](https://github.com/catppuccin/sddm)!

**Comment by prateektade** [comment]
You might already know, that we can directly apply the GTK theme to the GTK greeter by specifying the name in the `lightdm-gtk-greeter.conf` file. I'm not sure if the same can be done for the Slick greeter.

**Comment by syndrizzle** [comment]
The slick greeter does work with the gtk theme, it just doesn't really embrace the themes to its full potential.
